### DISCUSSION TITLE
Registry Data Creation

### DISCUSSION THREAD

**Discussion by ardihikaru** [bgsi-registry/OpenEHR-registry-template#2] [IMPORTANT]
By: Dr. Aly (@lamurian)
Ada beberapa lesson learned saat bikin template registri ini, please take notes supaya kita bisa mengatisipasi blocker saat dokter mulai membuat template.

1. Saat menggunakan [Archetype designer](https://tools.openehr.org/), error rate lebih rendah jika sign up menggunakan Google/Microsoft account
2. Gunakan GitHub untuk integrasi cloud repository, silakan sambungkan dengan repo berikut: [OpenEHR-registry-template](https://github.com/bgsi-registry/OpenEHR-registry-template)
3. Jika sudah terhubung dengan GitHub, maka lebih mudah untuk menggunakan ulang `Archetype` yang udah disiapkan
4. Secara desain, `Cluster` dalam archetype ngga bisa berdiri sendiri. `Cluster` harus digunakan sebagai ekstensi dari `Entry` (Read [this response](https://github.com/bgsi-registry/OpenEHR-registry-template) in GitHub).
5. Ketimbang membuat satu template besar dengan banyak Ad-hoc section, akan lebih efektif kalau memecah menjadi beberapa template kecil. Sebagai contoh, dalam `neurovascular.v0.opt` terdapat section:
- Administrative
- Clinical
- Clinical history
- Clinical score
- Demographic
- Laboratory
- Medication
- Physical examination
- Radiography

Secara total, saving and validating template besar membutuhkan waktu hampir 25 detik. Sementara per section umumnya hanya akan membutuhkan waktu <5 detik. Jadi ke depannya aku akan menyarankan untuk bikin multiple template/hubs. Aku pikir ini akan mempercepat progress dari tim FE juga karena kita bisa berfokus dengan small deliverables.

**Comment by mnardisasmita** [comment]
Ada hal yang mau ditanyakan ke dr @lamurian , untuk
![image](https://user-images.githubusercontent.com/30585827/165515890-c2d22695-6d23-4767-a4f0-1793ca085110.png)
 pembagian grup (template kecil) ini apakah ada standar templatenya atau memang atas keputusan sme nya untuk menempatkan variabel tersebut di grup nya, setelah melihat struktur dari form pdf asal nya?ff

**Reply by lamurian** [reply]
Ini adalah keputusan dari medical expert setelah melihat form secara keseluruhan dok

**Reply by mnardisasmita** [reply]
baik dok
